moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -300,16 +300,16 @@ const ComponentForm = ({
                 id="completion-date"
                 name="completionDate"
                 control={control}
-                render={({ onChange, value, ref }) => {
+                render={({ field }) => {
                   return (
                     <MobileDatePicker
-                      inputRef={ref}
-                      value={parseDate(value)}
+                      {...field}
+                      value={parseDate(field.value)}
```

Comment:
Since we have custom handling of the date, we can spread `field` which contains the `value` prop and then immediately override it with the custom date handling.